vendor/modules.txt

```diff
@@ -183,13 +183,14 @@ github.com/coreos/go-oidc
 # github.com/coreos/go-semver v0.3.0 => github.com/coreos/go-semver v0.3.0
 github.com/coreos/go-semver/semver
 # github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e => github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
-github.com/coreos/go-systemd/daemon
 github.com/coreos/go-systemd/dbus
 github.com/coreos/go-systemd/journal
```

Comment:
until we can also drop these two, I'm not sure depending on two versions of github.com/coreos/go-systemd is a net win

```
$ egrep -r 'go-systemd/[^v]' vendor | grep -v BUILD
vendor/go.etcd.io/etcd/pkg/systemd/journal.go:// Reference: https://github.com/coreos/go-systemd/blob/master/journal/journal.go.
vendor/go.etcd.io/etcd/pkg/logutil/zap_journal.go:	"github.com/coreos/go-systemd/journal"
vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/apply_systemd.go:	systemdDbus "github.com/coreos/go-systemd/dbus"
vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/apply_systemd.go:// NOTE: This function comes from package github.com/coreos/go-systemd/util
vendor/github.com/opencontainers/runc/libcontainer/cgroups/systemd/unified_hierarchy.go:	systemdDbus "github.com/coreos/go-systemd/dbus"
vendor/github.com/coreos/pkg/capnslog/journald_formatter.go:	"github.com/coreos/go-systemd/journal"
```

Reply:
The update of the etcd package would imply that etcd does not import `github.com/coreos/pkg` any more. To achieve this, we have to change the whole internal logging faccade inside etcd which is planned for 3.5.

The update of runc would require the whole runc project to switch to go modules. On the other hand we could try to remove the libcontainer dependency entirely, because this seems not optimal in my point of view.

Reply:
I'm not that familiar with libcontainer, so I'll defer to sig-node on what makes sense there. For etcd, switching to use versioned systemd seems reasonable. I do think we should work to get our dependencies in line so that we can actually *switch* to versioned systemd, instead of *adding* another systemd.

Reply:
I see, so both linked issue have to be resolved first, then we can switch over.
- etcd-io/etcd#11351
- opencontainers/runc#2171

Reply:
correct